data/src/test/java/org/oppia/android/data/backends/gae/api/ClassroomServiceTest.kt

```diff
@@ -47,6 +54,21 @@ class ClassroomServiceTest {
     assertThat(classroomResponse.body()!!.topicSummaryDicts?.get(0)?.name).isEqualTo("Math")
   }
 
+  @Test
+  fun testMockitoUnitTest() {
+    assertThat(mockLicenseFetcher.fetchText("https//:www.google.com")).contains("passed")
+  }
+
+  private fun initializeMockLicenseFetcher(): LicenseFetcher {
+    return mock {
+      on { fetchText(eq("https//:www.google.com")) } doReturn "passed"
+    }
+  }
+
+  interface LicenseFetcher {
+    fun fetchText(link: String): String
+  }
+
```

Comment:
I tried using capture but it kept throwing the same error. After a bit of research, I found that there is a separate library of [mockito for kotlin](https://github.com/mockito/mockito-kotlin), I added that, and it got fixed.

Reply:
Resolving as this is approved.